src/Core/Settings.h

```diff
@@ -849,6 +849,9 @@ class IColumn;
     M(Bool, print_pretty_type_names, true, "Print pretty type names in DESCRIBE query and toTypeName() function", 0) \
     M(Bool, create_table_empty_primary_key_by_default, false, "Allow to create *MergeTree tables with empty primary key when ORDER BY and PRIMARY KEY not specified", 0) \
     M(Bool, allow_named_collection_override_by_default, true, "Allow named collections' fields override by default.", 0)\
+    M(SQLSecurityType, default_normal_view_sql_security, ASTSQLSecurity::Type::INVOKER, "Allows to set a default value for SQL SECURITY option when creating a normal view.", 0) \
```

Comment:
This will require a double declaration of the same enum. Specifically for such cases, there exists `DECLARE_SETTING_ENUM_WITH_RENAME`, which I use. Here is another example: https://github.com/ClickHouse/ClickHouse/blob/1a14bd54db689c6d734c38b0185d242d00825c50/src/Core/Settings.h#L384